t/app/controller/waste_peterborough.t

```diff
@@ -502,6 +476,55 @@ FixMyStreet::override_config {
     };
 };
 
+FixMyStreet::override_config {
+    ALLOWED_COBRANDS => 'peterborough',
+    MAPIT_URL => 'http://mapit.uk/',
+    COBRAND_FEATURES => { bartec => { peterborough => {
+        url => 'http://example.org/',
+        auth_url => 'http://auth.example.org/',
+        sample_data => 1 } },
+        waste => { peterborough => 1 },
+        waste_features => {
+            peterborough => {
+                max_requests_per_day   => 3,
+                max_properties_per_day => 1,
+            },
+        },
+    },
+    STAGING_FLAGS => {
+        send_reports => 1,
+    },
+}, sub {
+    my ($b) = shared_bartec_mocks();
+
+    subtest 'test waste max-per-day' => sub {
+        SKIP: {
+            # MEMCACHED_HOST needs to be set to 'memcached.svc' in
+            # general.yml-example (or whatever config file you are using)
+            skip( "No memcached", 7 )
+                unless Memcached::set( 'waste-prop-test', 1 );
+            Memcached::delete("waste-prop-test");
+            Memcached::delete("waste-req-test");
+            $mech->get_ok('/waste/PE1 3NA:100090215480');
+            $mech->get_ok('/waste/PE1 3NA:100090215480');
+            $mech->get('/waste/PE1 3NA:100090215481');
+            is $mech->res->code, 403,
+                'Should be forbidden due to property limit';
+            $mech->content_contains('limited the number');
+            $mech->get('/waste/PE1 3NA:100090215480');
+            is $mech->res->code, 403,
+                'Should be forbidden due to overall view limit';
+            $mech->log_in_ok('staff@example.net');
+            $mech->get_ok(
+                '/waste/PE1 3NA:100090215480',
+                'Staff user shouldn\'t be limited'
+            );
+        }
+    };
+
+    $mech->log_in_ok('test@example.net');
```

Comment:
I think it would be preferable to log_out_ok here (or in fact, up a few lines at the end of the SKIP inside which you logged in).